src/Grids/Grids.jl

```diff
@@ -106,6 +106,11 @@ Grid topology for dimensions that are connected to other models or domains only
 """
 struct RightConnected <: AbstractTopology end
 
+topology_str(T) = string(T)
+topology_str(::Type{RightConnected}) = "RightConnected"
+topology_str(::Type{LeftConnected}) = "LeftConnected"
+topology_str(::Type{FullyConnected}) = "FullyConnected"
```

Comment:
```suggestion
topology_str(::Type{F}) where F = string(F)
```

Reply:
interesting. I don't think this is different, is it?

Reply:
oh i missed that you propose deleting lines

Reply:
this may indeed have the intended effect